CIP-0054/README.md

```diff
@@ -0,0 +1,227 @@
+---
+CIP: 54
+Title: Cardano Smart NFTs
+Authors: Kieran Simkin
+Comments-URI: https://forum.cardano.org/t/cip-draft-cardano-smart-nfts/100470
+Status: Draft
+Type: Standards
+Created: 2022-05-18
+License: CC-BY-4.0
+---
+# Cardano “Smart NFTs”
```

Comment:
Please no title in the markdown :pray:, the title already is part of the front-matter data.